app/src/main/res/layout/save_tab_group_button.xml

```diff
@@ -17,8 +17,10 @@
     android:text="@string/save_to_collection"
     android:textAllCaps="false"
     android:textColor="?contrastText"
+    android:fontFamily="@font/metropolis_medium"
     app:backgroundTint="?accent"
     app:icon="@drawable/ic_tab_collection"
     app:iconGravity="textStart"
     app:iconPadding="8dp"
+    app:fontFamily="@font/metropolis_medium"
```

Comment:
Some of these views have `fontFamily` declared twice, but others don't.  Are you sure we need this?